### ISSUE TITLE
JUnitAssertionsShouldIncludeMessage false-positives

### ISSUE THREAD

**Issue by dgroup** [dgroup/arch4u-pmd#135]
### Summary
| Key            |  Value |
|:---------------|:------|
| Rule           | JUnitAssertionsShouldIncludeMessage  |
| arch4u version       | 0.1.1 |

### Your rule definition
```xml
default configuration from pmd-java:6.44.0
```

### Describe the defect or false/positive rule violation

- [ ] Case 1: Class without test, but with @Before annotation on method
```java
class Base {    // false-positive violation here
     @Before
     public void setUp(){
            // ...
     }
}

class SomeTest extends Base {
     @Test
     public void justTestIt(){
          // ...
     }
}
```
